### DISCUSSION TITLE
Adjust amount of logs outputted

### DISCUSSION THREAD

**Discussion by 64-bitman** [ghostty-org/ghostty#3804] [Feature Requests, Ideas]
Is there a way to configure the amount of logs being shown, ex. configure a minimum log level to be displayed.

Thanks,

**Comment by jcollie** [comment]
There's no method to minimize it any further if you're running a release build (built with `ReleaseFast`).

**Reply by 64-bitman** [reply]
Are there any plans to add an option to minimize logs in the future?

**Reply by jcollie** [reply]
Nothing formal. If there are specific logs being emitted that you find excessive I'd start a GH discussion to see if that particular log can be omitted for release builds.

**Reply by 64-bitman** [reply]
> Nothing formal. If there are specific logs being emitted that you find excessive I'd start a GH discussion to see if that particular log can be omitted for release builds.

There arent any particular logs I want blocked, it's just that ghostty spams my entire system journal with warnings and info logs
![image](https://github.com/user-attachments/assets/cac20f61-def2-48f8-b550-18f42c7fc3cc)

**Comment by segln** [comment]
This is very annoying. Adjusting the log level or at least specify the log file path is required.
